docs/deployment_patterns/how_to_run_a_checkpoint_in_airflow.md

```diff
@@ -14,59 +14,254 @@ This guide will help you run a Great Expectations checkpoint in Apache Airflow,
+                "class_name": "Datasource",
+            }
+        },
+        "config_variables_file_path": os.path.join(
+            ge_root_dir, "uncommitted", "config_variables.yml"
+        ),
+        "stores": {
+            "expectations_store": {
+                "class_name": "ExpectationsStore",
+                "store_backend": {
+                    "class_name": "TupleFilesystemStoreBackend",
+                    "base_directory": os.path.join(ge_root_dir, "expectations"),
+                },
+            },
+            "validations_store": {
+                "class_name": "ValidationsStore",
+                "store_backend": {
+                    "class_name": "TupleFilesystemStoreBackend",
+                    "base_directory": os.path.join(
+                        ge_root_dir, "uncommitted", "validations"
+                    ),
+                },
+            },
+            "evaluation_parameter_store": {"class_name": "EvaluationParameterStore"},
+            "checkpoint_store": {
+                "class_name": "CheckpointStore",
+                "store_backend": {
+                    "class_name": "TupleFilesystemStoreBackend",
+                    "suppress_store_backend_id": True,
+                    "base_directory": os.path.join(ge_root_dir, "checkpoints"),
+                },
+            },
+        },
+        "expectations_store_name": "expectations_store",
+        "validations_store_name": "validations_store",
+        "evaluation_parameter_store_name": "evaluation_parameter_store",
+        "checkpoint_store_name": "checkpoint_store",
+        "data_docs_sites": {
+            "local_site": {
+                "class_name": "SiteBuilder",
+                "show_how_to_buttons": True,
+                "store_backend": {
+                    "class_name": "TupleFilesystemStoreBackend",
+                    "base_directory": os.path.join(
+                        ge_root_dir, "uncommitted", "data_docs", "local_site"
+                    ),
+                },
+                "site_index_builder": {"class_name": "DefaultSiteIndexBuilder"},
+            }
+        },
+        "anonymous_usage_statistics": {
+            "data_context_id": "abcdabcd-1111-2222-3333-abcdabcdabcd",
+            "enabled": False,
+        },
+        "notebooks": None,
+        "concurrency": {"enabled": False},
+    }
 )
 ```
 
-### Option 2: Running the `checkpoint script` output with a PythonOperator
+A `checkpoint_name` references a checkpoint in a series of subdirectories relative to the module (which is often the `great_expectations/checkpoints/` path), so that a `checkpoint_name = "taxi.pass.chk"`would reference the file `great_expectations/checkpoints/taxi/pass/chk.yml`. With a `checkpoint_name`, `checkpoint_kwargs` may be passed to the operator to specify additional, overwriting configurations. A `checkpoint_config` may be passed to the operator in place of a name. An example `checkpoint_config` is provided here:
```

Comment:
How much of the following code is specific to Airflow? Can we just show the operators and then link out to other GE docs for configuring checkpoint names and contexts?

Reply:
We can probably link some of this out to other docs, or better yet lines of code in the github repo or registry containing the example DAG this is from. It isn't totally Airflow-specific, but I wanted to include enough example code here that someone could reasonably set up an actual working DAG going through just this one page.